installing/installing_aws/installing-aws-government-region.adoc

```diff
@@ -41,6 +41,12 @@ include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
 
 include::modules/installation-initializing-manual.adoc[leveloffset=+1]
 include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional Resources
+
+* xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#nw-operator-cr-cno-object_installing-aws-network-customizations[Cluster Network Operator configuration object]
```

Comment:
this page should be removed from GovCloud page, and put it in Secret region page.

Reply:
@Wiharris I agree with @yunjiang29 that this cross reference does not apply to the GovCloud assembly. Furthermore, I suggest removing it entirely in favor a text-based cross reference near your new content regarding MTU. While the approach you take is valid, the `installation-configuration-parameters.adoc` module is large. Your cross reference will likely be overlooked because it is no where near your new content in the published out. In terms of proximity, I suggest a cross reference by your MTU text. For example:

"The maximum supported MTU in AWS Top Secret Region is not the same as AWS commercial. For more information about configuring MTU during isntallation, see the configuring the MTU during cluster install, see the  _Cluster Network Operator configuration object_ section in _Installing a cluster on AWS with network customizations_

I acknowledge the text-based xref is not ideal, but not being able to cross reference a module from another module is simply a limitation of of tool set that we have to work around.

Reply:
Hi Yunfei and Mike,

Thank you both so much for your help! I apologize for having this content in the wrong place and thank you both for clarifying! I will remove content from GovCloud page and place in Secret Region.

Reply:
Done! Doc preview link has been updated as well.